src/network/NetworkController.test.ts

```diff
@@ -27,161 +44,132 @@ describe('NetworkController', () => {
     });
   });
 
-  it('should throw when providerConfig property is accessed', () => {
-    const controller = new NetworkController();
-    expect(() => console.log(controller.providerConfig)).toThrow(
-      'Property only used for setting',
-    );
-  });
-
   it('should create a provider instance for default infura network', () => {
-    const testConfig = {
+    const networkControllerOpts = {
       infuraProjectId: 'foo',
+      messenger,
     };
-    const controller = new NetworkController(testConfig);
-    controller.providerConfig = {} as ProviderConfig;
+    const controller = new NetworkController(networkControllerOpts);
     controller.providerConfig = {} as ProviderConfig;
     expect(controller.provider instanceof Web3ProviderEngine).toBe(true);
   });
 
-  it('should create a provider instance for kovan infura network', () => {
-    const testConfig = {
+  const checkNetwork = (pType: NetworkType, expectIsCustomNetwork = false) => {
```

Comment:
What do you think about moving this to the top of the file so that it's easier to find?

Reply:
Also pulled it outside the describe, passing in messenger. Makes it more inline with the test helper pattern